cpp/include/cudf_test/column_utilities.hpp

```diff
@@ -199,8 +199,14 @@ template <>
 inline std::pair<thrust::host_vector<std::string>, std::vector<bitmask_type>> to_host(column_view c)
 {
   auto strings_data = cudf::strings::create_offsets(strings_column_view(c));
-  thrust::host_vector<char> h_chars(strings_data.first);
-  thrust::host_vector<size_type> h_offsets(strings_data.second);
+  thrust::host_vector<char> h_chars(strings_data.first.size());
+  thrust::host_vector<size_type> h_offsets(strings_data.second.size());
+  CUDA_TRY(
+    cudaMemcpy(h_chars.data(), strings_data.first.data(), h_chars.size(), cudaMemcpyDeviceToHost));
```

Comment:
These are synchronous memcpys. There should be a stream on which these are performed.

Reply:
The column utilities do not use streams right now.